lib/modules/datasource/terraform-module/index.ts

```diff
@@ -63,24 +62,16 @@ export class TerraformModuleDatasource extends TerraformDatasource {
 
     // Simplify response before caching and returning
     const dep: ReleaseResult = {
-      releases: res.versions.map((version) => ({
+      releases: res.modules[0].versions.map(({ version }) => ({
         version,
       })),
     };
-    if (res.source) {
-      dep.sourceUrl = res.source;
+    if (res.modules[0].source) {
+      dep.sourceUrl = res.modules[0].source;
```

Comment:
So I noticed that it is in the GitLab implementation, but not in the API doc, or the registry.terraform.io version - which is where i took the snapshots from.

It was hard to tell what this information is used for later, maybe better to completely remove if its not too important?

Reply:
i wouldn't ask if it's not important. so we probably should have two implementations. one for official registry with old API but more metadata and the new for other registries. 🤔

i also use stability days, so if we loose created timestamps it would be worsening.

@rarkins WDYT?

Reply:
Yea, maybe that makes sense, I can leave the old implementation, and if 404, try the new version. Ill add this back in today.